pkg/transport/generic_transport_controller.go

```diff
@@ -610,6 +610,9 @@ func cleanObject(object *unstructured.Unstructured) *unstructured.Unstructured {
 	delete(annotations, "kubectl.kubernetes.io/last-applied-configuration")
 	objectCopy.SetAnnotations(annotations)
 
+	// OCM removes the status only if subresource. We remove it in all cases.
```

Comment:
can we remove the OCM comment?
generic transport code doesn’t depend on ocm and shouldn’t depend.
it also shouldn’t care which transport is used, so specifying here ocm in the comments is confusing

Reply:
Changed the comment.